lib/webchat.yaml

```diff
@@ -1,12 +1,12 @@
 - base_path: /government/organisations/hm-passport-office/contact/passport-advice-and-complaints
-  open_url: https://hmpowebchat.klick2contact.com/v03/launcherV3.php?p=HMPO&d=717&ch=CH&psk=chat_a1&iid=STC&srbp=0&fcl=0&r=Chat&s=https://hmpowebchat.klick2contact.com/v03&u=&wo=&uh=&pid=2&iif=0
-  availability_url: https://hmpowebchat.klick2contact.com/v03/providers/HMPO/api/availability.php
+  open_url: https://omni.eckoh.uk/v03.5/launcherV3.php?p=HMPO&d=717&ch=CH&psk=chat_a1&iid=STC&srbp=0&fcl=0&r=Chat&s=https://omni.eckoh.uk/v03.5&u=&wo=&uh=&pid=2&iif=0
+  availability_url: https://omni.eckoh.uk/v03.5/providers/HMPO/api/availability.php
   open_url_redirect: false
```

Comment:
This [page](https://www.gov.uk/passport-advice-line) doesn't display webchat, just likes to a page that does, so I think this entire entry can be removed.